app/controllers/submission_controller.rb

```diff
@@ -29,11 +29,17 @@ def access_token
   end
 
   def payload
-    params.slice(
-      :meta,
-      :actions,
-      :submission,
-      :attachments
-    ).permit!
+    if params[:encrypted_submission]
```

Comment:
Even if is encrypted do you think is worth to add this param to be filtered in https://github.com/ministryofjustice/fb-submitter/blob/master/config/initializers/filter_parameter_logging.rb#L5 ?

Reply:
yeah probably unnecessary. i'll remove it from the filter 👍